_posts/2024-07-12-RSE-AUNZ-joins-OLS.md

```diff
@@ -31,12 +32,12 @@ If you want to join the community, sign-up to the [Google Groups mailing list](h
 
 ## RSE Asia Australia Unconference
 
-A major driver for RSE-AUNZ seeking fiscal sponsorship from OLS was the flagship event - [Research Software Engineer Asia Australia Unconference](https://rseaa.org), organised jointly with RSE Asia. This online event occurs every September and is attended by RSEs from many different research domains, all of whom work in some capacity with research software.
+A major driver for RSE-AUNZ seeking fiscal sponsorship from OLS was the flagship event - [Research Software Engineer Asia Australia Unconference](https://rseaa.org), organised jointly with [RSE Asia](https://rse-asia.github.io/RSE_Asia/). This online event occurs every September and is attended by RSEs from many different research domains, all of whom work in some capacity with research software.
 
-OLS support was crucial to run this event, since neither RSE associations have the necessary infrastructure to make financial transactions *necessary* for organising the Unconference.
+OLS support is crucial to run this event, since neither RSE associations have the necessary infrastructure to make financial transactions *necessary* for organising the Unconference.
 
 ## Fiscal Sponsorship with OLS
 
-Fiscal sponsorship is a relatively new program for OLS, mainly in response to the fact that people participating in the OLS mentorship programs ([Open Seeds](https://we-are-ols.org/openseeds/) and [Nebula](https://we-are-ols.org/nebula/)) often desire to take "next steps" for their communities or projects, but may not have the legal infrastructure to do so (e.g. ways to handle money, accounting, a registered organisation, etc.), or may have an institution they can work with that may not have the desired flexibility to meet their needs. 
+Fiscal sponsorship is a relatively new program for OLS. It was started mainly in response to the fact that people participating in the OLS mentorship programs ([Open Seeds]({% link openseeds/index.md %}) and [Nebula]({% link nebula/index.md %})) often desire to take "next steps" for their communities or projects, but may not have the legal infrastructure to do so (e.g. ways to handle money, accounting, a registered organisation, etc.), or may have an institution they can work with that may not have the desired flexibility to meet their needs. 
```

Comment:
I have an idea - we get a large cookie jar and put a dollar every time we don't use the `link` tag. :)

Reply:
We will get a lot of cookies at the end of the year 😋